pkg/cri/resource-manager/policy/builtin/balloons/balloons-policy.go

```diff
@@ -641,6 +670,9 @@ func (p *balloons) allocateBalloonOfDef(blnDef *BalloonDef, c cache.Container) (
 	if !blnDef.PreferSpreadingPods {
 		fillChain = append(fillChain, FillSamePod)
 	}
+	if blnDef.PreferSpreadingDifferentNamespaces {
+		fillChain = append(fillChain, FillSameNamespace, FillNewBalloon)
```

Comment:
Currently I preferred allowing two `FillNewBalloon`s in the chain rather than making this logic more complex. The reason is that it has no effect in functionality: either the first succeeds (and container goes to the new balloon instance) or both fail very quickly without side effects. The cost is that the chain looks uglier this way. Anyway, I thought this would be easier to follow than more complex building of the chain.

Reply:
K, just wanted to check. I think it's fine as is